app/forms/instance-create.tsx

```diff
@@ -669,6 +758,124 @@ const AdvancedAccordion = ({
             />
           )}
         </div>
+
+        <div className="flex flex-1 flex-col gap-4">
+          <h2 className="text-sans-md">
+            Floating IPs{' '}
+            <TipIcon>
+              Floating IPs are static IP addresses that can be attached to instances
```

Comment:
Might as well use the line we suffered for in the docs popover:

https://github.com/oxidecomputer/console/blob/9c519ce7ad81ae86a79bcf86f5293d779242f22f/app/pages/project/floating-ips/FloatingIpsPage.tsx?plain=1#L224